setup.sh

```diff
@@ -1,14 +1,18 @@
 # Once-off tasks to run when setting up the container
 # Tim Sutton, April 24, 2014
 # Install GeoGit
-cd /
-git clone http://github.com/boundlessgeo/GeoGit.git
-cd GeoGit/src/parent
+#cd /
+#git clone http://github.com/boundlessgeo/GeoGig.git
+#cd GeoGig/src/parent
+cd /GeoGig/src/parent
```

Comment:
This wont work if the geogig is not checked out already which will fail on the docker build service